python/demo/mixed-elasticity-sc/static-condensation-elasticity.py

```diff
@@ -0,0 +1,175 @@
+# Copyright (C) 2019 Michal Habera, Andreas Zilian
+#
+# This file is part of DOLFIN (https://www.fenicsproject.org)
+#
+# SPDX-License-Identifier:    LGPL-3.0-or-later
+
+# This demo solves a Cook's plane stress elasticity test in a mixed space
+# formulation. The test is a sloped cantilever under upward traction force
+# at free end. Static condensation of internal (stress) degrees-of-freedom
+# is demonstrated.
+
+import os
+
+import cffi
+import numpy
+import numba
+import numba.cffi_support
+from petsc4py import PETSc
+
+import dolfin
+import dolfin.cpp
+import dolfin.io
+import ufl
+
+
+filedir = os.path.dirname(__file__)
+infile = dolfin.io.XDMFFile(dolfin.MPI.comm_world,
+                            os.path.join(filedir, "cooks_tri_mesh.xdmf"))
+mesh = infile.read_mesh(dolfin.MPI.comm_world, dolfin.cpp.mesh.GhostMode.none)
+infile.close()
+
+# Se = stress element
+# Ue = displacement element
+Se = ufl.TensorElement("DG", mesh.ufl_cell(), 1, symmetry=True)
+Ue = ufl.VectorElement("CG", mesh.ufl_cell(), 2)
+
+S = dolfin.FunctionSpace(mesh, Se)
+U = dolfin.FunctionSpace(mesh, Ue)
+
+# Get local dofmap sizes for later local tensor tabulations
+Ssize = len(S.dofmap().cell_dofs(0))
```

Comment:
Use `FiniteElement::space_dimension` instead of `cell_dofs` (I haven't checked that it's been made available in Python yet.)

Also, move closer to where it's used.